it/uk/gov/hmrc/agentclientrelationships/controllers/RelationshipsControllerVATBehaviours.scala

```diff
@@ -110,32 +110,8 @@ trait RelationshipsControllerVATBehaviours { this: RelationshipsBaseControllerIS
         val result = doRequest
         result.status shouldBe 200
 
```

Comment:
this test case expected the creation of a relationship copy record even in the case of unknown agent details.
This no longer happens following the refactor so the expectation has been changed.
(The status returned is still 200, though, if there is an old relationship available)